src/main/resources/application.properties

```diff
@@ -76,10 +76,13 @@ warn.rbac.slow=true
 warn.rbac.tolerance=1S
 
 quarkus.jaeger.enabled=false
-quarkus.jaeger.service-name=policies-api 
+quarkus.jaeger.service-name=policies-api
 quarkus.jaeger.sampler-type=const
```

Comment:
With OpenTelemetry non of the `quarkus.jaeger.*` properties are needed or used in OpenTelemetry